Cognigy/WebchatWidget
# Improve focus color of close icon

This PR fixes color contrast issue for the focused state of the close window button in chat window header. 

Previous behavior: When the close button is focused, the close icon will get the darker shade of the primary color. This for some primary colors make the icon difficult to see, as just the 'X' changes color. 
Current behavior: Now, when the close icon gets focused, the background will get the dark primary color, instead of the icon.

To test: 
- Use keyboard to focus the close button in chat header.
- See if the close button is still clearly visible.
